File: portal/portal/App_Code/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace portal.App_Code
{
    public class Usuario
    {
        private long codigo;

        public long Codigo
        {
            get { return codigo; }
            set { codigo = value; }
        }
        private string nome;

        public string Nome
        {
            get { return nome; }
            set { nome = value; }
        }
        private string email;

        public string Email
        {
            get { return email; }
            set { email = value; }
        }
        private string senha;

        public string Senha
        {
            get { return senha; }
            set { senha = value; }
        }
        private TipoStatus status;

        public TipoStatus Status
        {
            get { return status; }
            set { status = value; }
        }

        public GrupoAcesso GrupoAcesso
        {
            get
            {
                return _grupoAcesso;
            }

            set
            {
                _grupoAcesso = value;
            }
        }

        public enum TipoStatus { Ativo, Inativo };

        private GrupoAcesso _grupoAcesso = new GrupoAcesso();

    }
}

File: portal/portal/App_Code/Usuario.php
<?php
###################################################################################################################
//USUÁRIO: Responsável pela área de acesso administrativa.
####################################################################################################################

namespace portal\App_Code {
require_once ("GrupoAcesso.php");
    class Usuario {
//ATRIBUTOS----------------------------------------------------------------------------------------------------------		
        private $codigo;
        private $nome;
        private $email;
        private $senha;
		private $TipoStatus;// 1 - Ativo, 0 - Inativo.
		private $codigo_grupo_acesso;
//CONSTRUTOR----------------------------------------------------------------------------------------------------------			
		public function __construct (){}
        
        public function setUsuario($codigo,$nome,$mail,$senha,$tipoStatus,$codigo_grupo_acesso){
			$this -> codigo = $codigo;
			$this -> nome = $nome;
			$this -> email = $mail;
			$this -> senha = $senha;
			$this -> TipoStatus = $tipoStatus;
			$this -> codigo_grupo_acesso = $codigo_grupo_acesso;
		}			
//SETS-----------------------------------------------------------------------------------------------------------------        
		public function setCodigo($codigo){
			$this -> codigo = $codigo;
        }
		public function setNome($nome){
			$this -> nome = $nome;
		}
		public function setEmail($mail){
			$this -> email = $mail;
		}
		public function setSenha($senha){
			$this -> senha = $senha;
		}		
        public function setTipoStatus($tipoStatus) {
			$this -> TipoStatus = $tipoStatus;
        }		
        public function setCodigoGrupoAcesso($codigo_grupo_acesso) {
			$this -> codigo_grupo_acesso = $codigo_grupo_acesso;
        }
//GETS-----------------------------------------------------------------------------------------------------------------			
		public function getCodigo(){
			return $this -> codigo;
        }
        public function getNome() {
			return $this -> nome;
        }
        public function getEmail() {
			return $this -> email;
        }
        public function getSenha() {
			return $this -> senha;
        }
        public function getTipoStatus() {
			return $this -> TipoStatus;
        }
        public function getCodigoGrupoAcesso() {
			return $this -> codigo_grupo_acesso;
        }
    }
}
?>